python/mxnet/ndarray.py

```diff
@@ -1352,6 +1378,12 @@ def divide(lhs, rhs):
     >>> (x/y).asnumpy()
     array([[ inf,  inf,  inf],
            [  1.,   1.,   1.]], dtype=float32)
+    >>> mx.nd.divide(x,y).asnumpy()
+    array([[ inf,  inf,  inf],
```

Comment:
Again - what is x and what is y? Again the examples need to be self-contained, how else can the user interpret the results. 

*Big Question* Why are you dividing by 0 in the example?? This is a bug - a user should never be dividing by 0. Is this just used as an example because it happens to be the same values of x and y as in the previous examples?